src/Microsoft.VisualStudio.Editors/Common/Utils.vb

```diff
@@ -1342,11 +1342,12 @@ Namespace Microsoft.VisualStudio.Editors.Common
         ''' <returns>True is the current Framework Profile is Client</returns>
         Friend Function IsClientFrameworkSubset(Hierarchy As IVsHierarchy) As Boolean
             Debug.Assert(Hierarchy IsNot Nothing, "Hierarchy is required")
-            Dim service As MultiTargetService = New MultiTargetService(Hierarchy, VSConstants.VSITEMID_ROOT, False)
-            ' AuthenticationService is present only in server frameworks. We want to test for presence of this type 
-            ' before enabling server-specific functionality
-            Return Not service.IsSupportedType(GetType(Web.ApplicationServices.AuthenticationService))
-
+            'TODO: Merged PIAs
+            'Dim service As MultiTargetService = New MultiTargetService(Hierarchy, VSConstants.VSITEMID_ROOT, False)
+            '' AuthenticationService is present only in server frameworks. We want to test for presence of this type 
+            '' before enabling server-specific functionality
+            'Return Not service.IsSupportedType(GetType(Web.ApplicationServices.AuthenticationService))
+            Return True
```

Comment:
Can we log an issue against 17.0 to make sure we don't forget to revert all the occurrences of `TODO: Merged PIAs` in the code?

Reply:
Won't be able to forget since the PR won't merge without these :)